ui-modules/blueprint-composer/app/components/catalog-selector/catalog-selector.directive.js

```diff
@@ -127,15 +169,15 @@ function controller($scope, $element, $q, $uibModal, $log, $templateCache, palet
     $scope.getPlaceHolder = function () {
         return 'Search';
     };
-
+    
     $scope.isLoading = true;
 
     $scope.$watch('search', () => {
         $scope.freeFormTile = {
             symbolicName: $scope.search,
             name: $scope.search,
             displayName: $scope.search,
-            supertypes: [$scope.family.superType]
+            supertypes: ($scope.family ? [ $scope.family.superType ] : []),
```

Comment:
`$scope.family` is marked as required so this should not be needed. Callers of the `catalog-selector` directive should always pass it

Reply:
can be passed in as null however